docs/dev/stream/state.md

```diff
@@ -429,3 +429,120 @@ public static class CounterSource
 
 Some operators might need the information when a checkpoint is fully acknowledged by Flink to communicate that with the outside world. In this case see the `org.apache.flink.runtime.state.CheckpointListener` interface.
 
+## Custom serialization for managed state
+
+This section is targeted as a guideline for users who require using custom serialization for their state, covering how
```

Comment:
This section is targeted as a guideline for users who require custom serialization for their state

"using" doesn't really work in this context. You could say "the use of" instead, but it doesn't add anything.